scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2081,15 +2086,20 @@ class FormatOps(
     }
 
     object BlockImpl extends Impl {
+
       def tryGetSplits(ft: FormatToken, nft: FormatToken)(implicit
-          style: ScalafmtConfig
+          style: ScalafmtConfig,
+          line: sourcecode.Line
       ): Option[Seq[Split]] = {
         val leftOwner = ft.meta.leftOwner
         findTreeWithParentSimple(nft.meta.rightOwner)(_ eq leftOwner) match {
           case Some(t: Term.Block)
-              if t.stats.lengthCompare(1) > 0 && isBlockStart(t, nft) =>
+              if (t.stats.lengthCompare(1) > 0 ||
+                getBlockSingleStat(t).exists(_.isNot[Term]))
+                && isBlockStart(t, nft) =>
             Some(getSplitsMaybeBlock(ft, nft, t, true))
-          case _ => None
+          case other =>
```

Comment:
must be debug leftover. could you please restore two lines.

Reply:
fixed